tests/vision/test_orientation.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for orientation class."""
+
+from numpy import allclose
+from pyquaternion import Quaternion
+
+from j5.vision.orientation import Orientation
+
+
+def test_quaternion() -> None:

```

Comment:
Could you update these function names to match the convention we have in the rest of the tests?

For example: `def test_coordinate_from_cylindrical() -> None:`

Therefore, this one would be:

```suggestion
def test_orientation_quaternion() -> None:

```